docs/getting-started/user-guides/linux.md

```diff
@@ -248,7 +248,7 @@ Decred. Open a third terminal and go to your Decred directory. Type in
 (once again using your values for the square brackets):
 
 ```no-highlight
-~/Decred/$ ./dcrctl -u [daemon username] -P [daemon password] --wallet walletpassphrase [wallet password] 0
+~/Decred/$ ./dcrctl -u [wallet username] -P [wallet password] --wallet walletpassphrase [wallet password] 0
 ```
 
 What we are doing here is authenticating with the daemon and using the
```

Comment:
Also could you use a more descriptive commit message?

Something like "Fix dcrctl command example for linux user-guide" or anything that tells use what is in the commit/